docs/install/windows-release-channels.md

```diff
@@ -0,0 +1,320 @@
+# Switching Netdata Install Types and Release Channels on Windows
+
+You can switch between different Netdata release channels on Windows based on your needs. This guide covers the process with step-by-step instructions.
+
+## Understanding Windows Installation Architecture
+
+Unlike Linux systems where Netdata has multiple install types (native packages, static builds, Docker, etc.), **Windows only has one install type**: the **MSI installer**.
+
+This simplifies the switching process significantly:
+- No need to determine your "install type" (it's always MSI)
+- No complex switching procedures between different installation methods
+- MSI handles upgrades automatically
+
+### Release Channels Available
+
+| Channel | Download URL | Update Frequency | Recommended For |
+|---------|-------------|------------------|-----------------|
+| **Stable** | `https://github.com/netdata/netdata/releases/latest/download/netdata-x64.msi` | Major releases | Production systems, most users |
+| **Nightly** | `https://github.com/netdata/netdata-nightlies/releases/latest/download/netdata-x64.msi` | Daily builds | Testing, early adopters, bleeding-edge features |
+
+## Data Preservation When Switching Channels
+
+When switching between release channels on Windows, the MSI installer automatically preserves your important data and configuration.
+
+**Preserved during channel switches:**
+- Configuration files (`netdata.conf`, collector configs)
```

Comment:
I suggest to add the path (C:\Program Files\Netdata\etc\netdata) and not only mention the file name.